addons/website_event_questions/i18n/website_event_questions.pot

```diff
@@ -7,7 +7,7 @@ msgstr ""
 "Project-Id-Version: Odoo Server saas~14.5\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-09-14 10:29+0000\n"
-"PO-Revision-Date: 2021-09-14 10:29+0000\n"
+"PO-Revision-Date: 2023-11-14 13:11+0000\n"
```

Comment:
We generally don't change that (minimal diff = add new strings and that's all :) )